about.md

```diff
@@ -27,4 +31,4 @@ FOSSASIA Organization Team
 ####Hong Phuc Dang
 FOSSASIA Organizer
 
-[hp@fossasia.org](mailto:hp@fossasia.org)
\ No newline at end of file
+[hp@fossasia.org](mailto:hp@fossasia.org)
```

Comment:
Don't delete and replace same stuff, if possible.

Reply:
These changes are generally non-intentional and occur due to extra spacing. So It is okay! :)